pennylane/operation.py

```diff
@@ -19,21 +19,17 @@
 Description
 -----------
 
-Qubit Operations
+Qubit Operators
 ~~~~~~~~~~~~~~~~
```

Comment:
@josh146 I think I need help rewriting this (since it contains differentiation info I don't understand), or shall we delete it? The information should be rather provided in the classes and the dev guide?

Reply:
I think I would delete everything from "Most gates support the analytic method,..." to the end of the subsection 🙂 

The first two paragraphs are still quite useful, as they introduce the concept of 'trainable parameters'